dolphinscheduler-master/src/main/java/org/apache/dolphinscheduler/server/master/runner/queue/DelayEntry.java

```diff
@@ -54,14 +54,6 @@ public long getDelay(@NotNull TimeUnit unit) {
     @Override
     public int compareTo(@NotNull Delayed o) {
         DelayEntry<V> other = (DelayEntry<V>) o;
-        int delayTimeMillsCompareResult = Long.compare(delayTimeMills, other.delayTimeMills);
-        if (delayTimeMillsCompareResult != 0) {
-            return delayTimeMillsCompareResult;
-        }
-
-        if (data == null || other.data == null) {
-            return 0;
-        }
```

Comment:
Don't do this kind of change, this class is `DelayEntry` if you remove the compare logic this class cannot work